deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml

```diff
@@ -23,6 +23,9 @@ spec:
         - ./grafanadi
         - --config-file=/app/config-file.yaml
         - --enable-kube-client={{ .Values.grafanadiInitKubeClient }}
+        - --log_dir=/logs
```

Comment:
Could you please put these parameters in values.yaml?

Reply:
done